pages/1_14/getting_started/code_structure.md

```diff
@@ -24,11 +16,25 @@ While it often doesn't contain much code it is the main
 entry point of your mod.
 
 ```java
-@Mod(Reference.MODID)
+package com.example.examplemod;
+
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static com.example.examplemod.ExampleMod.MODID;
+
+@Mod(MODID)
 public class ExampleMod {
 
     public static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Shouldn't this be gotten from the preinit event?

Reply:
Oh actually forgot this is 1.14 heh, lemme know if this has actually changed and this is the way to do it.